.changeset/thick-streets-begin.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': patch
+---
+
+Fixed memory timeline event markers failing to build with current Recharts types.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)